### ISSUE TITLE
Jetty-12 ee9-annotations module does not work with JPMS

### ISSUE THREAD

**Issue by lachlan-roberts** [eclipse/jetty.project#9909]
**Jetty version(s)**
12

**Description**
```
Error occurred during initialization of boot layer
java.lang.module.FindException: Module jakarta.cdi not found, required by jakarta.transaction
```

**How to reproduce?**
```
java -jar $JETTY_HOME/start.jar --add-modules=ee9-annotations
java -jar $JETTY_HOME/start.jar --jpms
```

**Comment by janbartel** [comment]
@lachlan-roberts our distribution tests for jpms have been disabled for ee8/9/10 so no doubt there could be lots of heretofore undiscovered issues.

**Comment by janbartel** [comment]
closed after fix #9911
